scripts/merge-PR.sh

```diff
@@ -16,6 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# Setting the script to fail if anything goes wrong
+set -e
 
 # Use this to simplify the rebasing of PRs. PRs will be rebased during the merge on this process.
 # use: ./rebase-PR <PRID> textual description
```

Comment:
ahhhhh.. ok... 

It used to be called rebase-PR when it was on my ~home/bin only, before I sent as a commit :)

thanks for pointing it out.